scripts/ceph-windows/run_tests

```diff
@@ -68,16 +68,16 @@ if [[ -f $LOCAL_SCRIPT_PATH ]]; then
     scp_upload $WIN_WORKUNITS_DIR /workspace/workunits
     SSH_TIMEOUT=30m ssh_exec powershell.exe -File /workspace/workunits/run-tests.ps1
 else
```

Comment:
This `else` branch is currently only serving quincy, right?  If so, can you add a comment here stating that so that it's clear that this is legacy not-to-be-touched stuff.